assets/i18n/en.toml

```diff
@@ -4,26 +4,26 @@ docs = "Documentation"
 blog = "Blog"
 github = "GitHub"
 login = "Vehicle logins"
-about = "About evcc"
+about = "About"
 
 [header.theme]
 auto = "Design: system"
 light = "Design: light"
 dark = "Design: dark"
 
 [footer.version]
-availableShort = "update"
-availableLong = "update available"
-modalTitle = "Update available"
-modalUpdateStarted = "Evcc will restart after the update.."
-modalInstalledVersion = "Currently installed version"
-modalNoReleaseNotes = "No release notes available. More information about the new version can be found here:"
+availableShort = "install"
```

Comment:
The idea is to avoid "update" about something that is an `upgrade`, since the latter is less reversible, and potentially breaks things.

Reply:
Why don't we call it `upgrade` then?

Reply:
Then someone might translate it to ~"update", or ~"improve something".

Reply:
I would leave it "update" as this is general nature for this.
With some help from google

> Basically, think of an upgrade as a less frequent, more drastic change to the software you are currently using. A software update, on the other hand, can be more frequent, fix little bugs or make small tweaks, and is often used to repair the product. This sums up the main difference between system updates.

Thus evcc is updated and upgraded on the other hand. I do feel that evcc is more updated and enhanced and less frequent upgraded (like new cars, hardware support)?

Reply:
@int5749 The word for that is "flashed", to do with ~"firmware".
Pulling down information about what files are new is an update.
Pressing F5 to refresh and similar is an update.
Downloading new software with new versions is a system upgrade.